test/runtests.jl

```diff
@@ -39,6 +39,12 @@ end
     include("butensor.jl")
 end
 
+# note: OMEinsumContractionOrders should not be loaded before this point
+# as there is a test which requires it to be loaded after
+@testset "OMEinsumOptimizer extension" begin
+    include("omeinsumcontractionordres.jl")
+end
```

Comment:
Sure, I add this in tests and change the lowest support version back to `v1.8`

Reply:
Ops, in CI for `v1.8`, `Unsatisfiable requirements detected for package OMEinsumContractionOrders for `v1.8`` since `OMEinsunContractionOrders` only support `v1.9` or higher.